python/ctranslate2/converters/transformers.py

```diff
@@ -111,8 +111,7 @@ def _load(self):
             model = self.load_model(model_class, self._model_name_or_path, **kwargs)
 
             tokenizer = self.load_tokenizer(
-                tokenizer_class,
-                self._model_name_or_path,
+                tokenizer_class, self._model_name_or_path, **kwargs
```

Comment:
`kwargs` contains arguments that not related to the tokenizer. Can you just set `trust_remote_code`?

Reply:
Sure

Reply:
Have made the change Please review again